### ISSUE TITLE
UTF-8 ACP seems to hork the Terminal store app forever? "The parameter is incorrect"

### ISSUE THREAD

**Issue by aliozgur** [microsoft/terminal#3039]
# Environment

```none
Windows build number: 10.0.18362.0
Windows Terminal version: v0.5.2681.0
```

# Steps to reproduce
Find the Windows  Terminal (Preview) app and click.

# Expected behavior
Windows Terminal should launch.

# Actual behavior

![image](https://user-images.githubusercontent.com/799194/66106817-796dc380-e5c7-11e9-8a69-6d181b916ab8.png)

**Comment by aliozgur** [comment]
This problem popped up after I turned on the "Use Unicode UTF-8 for worldwide lanugage support". 
![image](https://user-images.githubusercontent.com/799194/66107056-07e24500-e5c8-11e9-9701-38a918542186.png)

The interesting thing is, the problem still exists even if I turned of the "Use Unicode UTF-8 for worldwide lanugage support" feature.

**Comment by DHowett-MSFT** [comment]
So wait -- you turned on this feature, rebooted, the problem began, and now you've turned it off and rebooted again _and it still fails_? The globalization team (who owns the UTF-8 ACP feature(!)) is going to find this extremely interesting. Thanks!

**Comment by aliozgur** [comment]
Yes that’s exactly the case. Turned on the feature, restarted Windows and terminal failed. Turned off the feature, restarted Windows again and terminal still fails.

**Comment by aliozgur** [comment]
This happende for another Microsoft AppStore app Trello as well. 

First I got this error (A different one)
C:\Program Files\WindowsApps\45273LiamForsyth.PawsforTrello_2.11.1.0_x64__7pb5ddty8z1pa\app\Trello.exe
The data area passed to a system call is too small.

On subsequent attempts to run the app again I got "The parameter is incorrect" error.

**Comment by aliozgur** [comment]
Ok, I think I've resolved the problem. Here are the steps I've followed to resolve the issue

* I've noticed some ESENT related errors in Event Log
* Then I Googled for the logged ESENT errors. 
* I could not find any direct solution for those errors, but some guys advised to update SQL Server
* Then I updaded my local SQL Server 2014 to lates service pack (SP3) and then installed the latest Cummulative Update (CU4)

After couple of restarts Widows Terminal and the other app (Trello) seems to work.
